recipes/ckanweb-deploy.rb

```diff
@@ -89,21 +90,28 @@
 	execute "Install CKAN #{version}" do
 		user "#{service_name}"
 		group "#{service_name}"
-		command "#{pip} install -e 'git+#{apprelease}#egg=#{service_name}'"
+		command "#{pip} install -e 'git+#{apprelease}@#{version}#egg=#{service_name}'"
```

Comment:
This will break if `version` remains empty (ie no revision specified and no version in the source URL). Should we add a fallback for `version` to default to qgov-master?

Reply:
above adds master, can you update to not insert version and skip if version is empty. but satying that we should always have a version to ingest.

Reply:
Actually I just realised that this needs some further tweaking; the previous change had side effects (`apprelease` was supposed to be cut down to a plain URL but now would not be if `version` is already populated). Will push another commit soon.